x/rewards/keeper/flat_fee.go

```diff
@@ -37,3 +40,36 @@ func (k Keeper) GetFlatFee(ctx sdk.Context, contractAddr sdk.AccAddress) (sdk.Co
 
 	return fee, true
 }
+
+// ImportFlatFees initializes state from the genesis flat fees data.
+func (k Keeper) ImportFlatFees(ctx sdk.Context, flatFees []types.FlatFee) {
+	for _, flatFee := range flatFees {
+		if err := k.SetFlatFee(ctx, flatFee.MustGetContractAddress(), flatFee.GetFlatFee()); err != nil {
+			panic(fmt.Sprintf("flat fee: %+v is invalid: %s", flatFee, err))
+		}
+	}
+}
+
+// ExportFlatFees returns the flat fees genesis data for the state.
+func (k Keeper) ExportFlatFees(ctx sdk.Context) []types.FlatFee {
+	store := prefix.NewStore(k.state.FlatFee(ctx).stateStore, types.FlatFeePrefix)
+
+	iterator := store.Iterator(nil, nil)
+	defer iterator.Close()
+
+	var fees = make([]types.FlatFee, 0)
+	for ; iterator.Valid(); iterator.Next() {
+		contractAddr := sdk.AccAddress(iterator.Key())
+		coin, ok := k.state.FlatFee(ctx).GetFee(contractAddr)
+		if !ok {
+			panic(fmt.Sprintf("invalid coin: %+v for address: %s", coin, contractAddr))
+		}
```

Comment:
```suggestion
		coinBz := iterator.Value()
		var coin sdk.Coin
	    cdc.MustUnmarshal(coinBz, &coin)
```
You dont need to call the `GetFee` func to get the flat fee value. 
Should be able to access the value from `iterator.Value()` and then unmarshal it into a sdk.Coin obj.

Reply:
Addressed with 443a2e1046de7794e197029dd9b277028ae55c51